sql/mob_spawn_points.sql

```diff
@@ -11938,13 +11938,13 @@ INSERT INTO `mob_spawn_points` VALUES (16990419,'Chigoes_Nit','Chigoe\'s Nit',48
 INSERT INTO `mob_spawn_points` VALUES (16990420,'Chigoes_Nit','Chigoe\'s Nit',48,178.088,-18.427,314.379,81);
 INSERT INTO `mob_spawn_points` VALUES (16990421,'Chigoes_Nit','Chigoe\'s Nit',48,178.205,-18.913,313.661,90);
 INSERT INTO `mob_spawn_points` VALUES (16990422,'Chigoes_Nit','Chigoe\'s Nit',48,178.418,-18.190,313.134,104);
-INSERT INTO `mob_spawn_points` VALUES (16990423,'Dark_Rider','Dark Rider',49,549.927,-10.000,171.641,221);
-INSERT INTO `mob_spawn_points` VALUES (16990424,'Dark_Bugler','Dark Bugler',50,59.596,-19.063,255.521,96);
-INSERT INTO `mob_spawn_points` VALUES (16990425,'Dark_Bugler','Dark Bugler',50,175.097,-18.686,310.768,88);
-INSERT INTO `mob_spawn_points` VALUES (16990426,'Dark_Bugler','Dark Bugler',50,281.659,-17.688,248.399,65);
-INSERT INTO `mob_spawn_points` VALUES (16990427,'Dark_Esquire','Dark Esquire',51,330.384,-9.198,223.755,246);
-INSERT INTO `mob_spawn_points` VALUES (16990428,'Dark_Esquire','Dark Esquire',51,450.198,-16.331,300.446,125);
-INSERT INTO `mob_spawn_points` VALUES (16990429,'Dark_Esquire','Dark Esquire',51,538.488,-11.782,166.673,240);
+INSERT INTO `mob_spawn_points` VALUES (16990423,'Dark_Rider','Dark Rider',49,-162,-20,-494,221);
```

Comment:
So does he always spawn at one position, but goes onto different pathing routes from that spawn point?

Reply:
No - I have defined at least 3 random spawn points in each zone and from those spawn points he will path - the warhorse hoofprint will then spawn somewhere in that path he took.